xmhf/xmhf/src/xmhf-core/xmhf-secureloader/arch/x86/sl-x86.c

```diff
@@ -178,6 +178,11 @@ void xmhf_sl_arch_sanitize_post_launch(void){
         printf("\nSL: Restoring mtrrs...");
         
         restore_mtrrs(&(os_mle_data->saved_mtrr_state));
+        
+        /* always set the TXT.CMD.SECRETS flag */
+        write_priv_config_reg(TXTCR_CMD_SECRETS, 0x01);
+        read_priv_config_reg(TXTCR_E2STS);   /* just a fence, so ignore return */
+        printf("SL: set TXT.CMD.SECRETS flag\n");
```

Comment:
This should be moved to hypapps/trustvisor/src/appmain.c since it achieve a TrustVisor hypapp specific property

Reply:
Done, looks like `write_priv_config_reg` can also be accessed from XMHF runtime and hypapps.